### DISCUSSION TITLE
Is it possible to add the feature of commonly used receiving devices to the send page?

### DISCUSSION THREAD

**Discussion by oyster-zzz** [localsend/localsend#727] [Discussion (English, International)]
It is hoped that a feature can be added to the send page that allows the user to quickly select some commonly used receiving devices to connect to, which would simplify the process of having to enter a fixed IP address each time. Looking forward to your reply!

**Comment by KenChen1998** [comment]
For example, when I want to transfer something through wireguard vpn network, those devices are not shown on the list . Therefore,  I appreciate if those fixed ip deviced could be saved on the list. 
![image](https://github.com/localsend/localsend/assets/53257544/57725927-d0ac-4f96-9762-45572f73904e)
